src/test/java/org/jenkinsci/plugins/ssegateway/EventHistoryStoreTest.java

```diff
@@ -135,13 +138,24 @@ public void test_autoDeleteOnExpire() throws Exception {
             // be a bit flaky though, so lets just make sure some of the messages
             // have expired, but not all.
             Assert.assertTrue(EventHistoryStore.getChannelEventCount("job") > 0);
+            // The Windows machines on ci.jenkins.io need extra time.  Assumed due
+            // to a slower file system or slower storage.
+            if (Functions.isWindows() && System.getenv("CI") != null) {
+                waitTimer.waitUntil(13500); // 13.5 sec = 4.5 + 9 seconds cushion
```

Comment:
why not simply remove when running on windows and CI? 
by experience, this sort of number approximation turns to result into flaky tests.

Reply:
Sure.  Happy to do that as well.

Reply:
Pushed as 6453ddf1a29b692d0ee6e6cb38362f079b69950d